Self-study Q&A.md

```diff
@@ -0,0 +1,16 @@
+<!-- What is the DOM? -->
+An interface that allows a programming language to manipulate the content, structure, and style of a website.
+
+<!-- What is an event? -->
+Something you do on the web page, like moving your mouse around, scrolling, or clicking a link
+
+<!-- What is an event listener? -->
+A procedure or function in a computer program that waits for an event to occur and in effect since then
+
+<!-- Why would we convert a NodeList into an Array? -->
+To use methods like forEach, filter, map?
```

Comment:
Yes even a thought a Node list does look similar to an array NodeLists are a way to access DOM elements, and Arrays are a JavaScript object you can use to contain collections of stuff. and if we convert one into an array we would get access to those array methods like you mention